.github/workflows/ci-stable-issue-pr.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
```

Comment:
I think the stale message should be

"The issue had no activity for 30 days, mark with Stale label."